internal/cri/server/service.go

```diff
@@ -155,6 +155,9 @@ type criService struct {
 	sandboxService sandboxService
 	// runtimeHandlers contains runtime handler info
 	runtimeHandlers []*runtime.RuntimeHandler
+	// containerStoreAdded indicates whether container is already added in containerStore
+	// when restart containerd
+	containerStoreAdded chan struct{}
```

Comment:
why not bool / atomicBool ?

Reply:
If it is bool ,I have to write a for  func to check it turn to true.